### ISSUE TITLE
clean up unused files

### ISSUE THREAD

**Issue by proppy** [efabless/caravel_user_project#132]
is https://github.com/efabless/caravel_user_project/blob/main/verilog/rtl/uprj_netlists.v still used?

**Comment by proppy** [comment]
similarly are files in https://github.com/efabless/caravel_user_project/tree/main/verilog/includes still used? (they don't seem to be using a consistent syntax)

**Comment by marwaneltoukhy** [comment]
> similarly are files in https://github.com/efabless/caravel_user_project/tree/main/verilog/includes still used? (they don't seem to be using a consistent syntax)

Yes they are used in simulation, they have inconsistent syntax because GL and RTL simulations are done using iVerilog, while GL+SDF simulations are done using CVC. The tools use different syntax for including files.

**Comment by proppy** [comment]
@marwaneltoukhy is  https://github.com/efabless/caravel_user_project/blob/main/verilog/rtl/uprj_netlists.v also used in simulation?

**Comment by marwaneltoukhy** [comment]
@proppy No it's not, at least not the default flow. Investigating if some users might be using it for any other reason, I don't want to remove a file that might end up breaking someone's flow.
